webapp/channels/src/actions/websocket_actions.jsx

```diff
@@ -238,6 +239,7 @@ export function reconnect() {
         }
 
         dispatch(loadChannelsForCurrentUser());
+        dispatch(fetchAllMyTeamsChannels());
```

Comment:
With this, can we get rid of the call to `loadChannelsForCurrentUser` above? It seems like we're loading the same channels twice now

Reply:
I think the above one also loads DMs/GMs? It calls `loadNewDMIfNeeded` and `loadNewGMIfNeeded` per channel loaded, I don't think `fetchAllMyTeamsChannels` does that.